app/components/welcome-page-copy.css

```diff
@@ -2,26 +2,21 @@
 https://github.com/ember-cli/ember-welcome-page/blob/main/ember-welcome-page/src/components/welcome-page.css
 used to easily set up this demo. */
 
-@value ember-orange from 'css-modules-to-scoped-css/styles/app';
-
-#ember-welcome-page-id-selector, .main {
+#ember-welcome-page-id-selector {
   padding: 2em;
   border-style: solid;
   border-width: 1px;
 }
 @media (max-width: 700px) {
-  #ember-welcome-page-id-selector, .main {
```

Comment:
`#ember-welcome-page-id-selector` and its styles initially come from `ember-welcome-page` addon. To make it work with ember-css-modules, I added a class `.main`, because the id selector is transformed only CSS-side, but not in the DOM: that mismatch prevents the styles from applying.

In ember-scoped-css, the id selectors are not transformed, so the extra `.main` class is not needed.